### ISSUE TITLE
Any interest in edit mode as well?

### ISSUE THREAD

**Issue by travisterrell** [WiseGuru/obsidian-automatic-list-styles#4]
Hi, I just discovered this very useful plugin and greatly appreciate it. I'm wondering if you've ever toyed with the idea of making it function in the Edit mode as well? Reading mode is pretty, but I find myself in Edit mode 95% of the time because the formatting is generally _good enough_ and I have a habit of endlessly editing things.

Admittedly, I have not yet studied the code or tried to do it for myself, so I have no idea how feasible it would be with your implementation. It might be applied in a way that could realistically only ever work for the Reading view. Nonetheless, I figured it was worth an ask!

Thanks,
Travis

**Comment by WiseGuru** [comment]
I would love to do that, but I haven't been able to figure it out. I'm also not a developer, so that's not a surprise haha

I occasionally come back to it to try again, but if you want to take a look, here are a couple tips:
1. It's easier to edit just the .ts file and work back from there once you get something working.
2. Using CSS snippets might also be more efficient and easier to troubleshoot, I'm just not as familiar with them.

Here are some links that might help you get started with CSS editing (and that I'm saving for posterity):
1. https://forum.obsidian.md/t/getting-comfortable-with-obsidian-css/133/6
2. https://forum.obsidian.md/t/obsidian-css-inspector-workflow/58178

I dunno, if you find anything interesting, I would love to know!

**Comment by travisterrell** [comment]
Thank you for the response! Yeah, I might just have to give that a look soon. Having recently moved from other notetaking apps, I find myself constantly desiring different list styles in different situations. 
It's pretty cool that you created and shared this, despite not being a developer! I actually am a dev, but I'm extremely unfamiliar with Obsidian, so I greatly appreciate you sharing the tips and resources! I'll let you know if I dig into it, and will share my findings if don't come up with a solution.

Thanks again!

**Comment by willem-schlieter** [comment]
Hi @WiseGuru, I appreciate this nice plugin, used it for a while! But I also missed this functionality in Edit Mode! (@travisterrell 95% Edit Mode user as well...) So I am currently developing a plugin that supports Edit Mode as well:

[https://github.com/willem-schlieter/lawlist](https://github.com/willem-schlieter/lawlist)

Feel free to have a look, it works fine in Edit mode, just needs to be downloaded from GitHub and compiled locally, as its not published to Obsidian yet. Read Mode (and thus, which is important for me, PDF export) is not yet supported, but I am planning to implement that soon.

I saw you are using CSS `list-style-type` in your plugin. I thought about that as well for Read Mode, but it doesn't support custom styles apart from the standard variants, does it? As you already worked on Read Mode: Any idea how to accomplish full customization there as well?

**Comment by WiseGuru** [comment]
@willem-schlieter  This is great! Just tested it and it's awesome. I would advise adding an instruction to run `npm install` to make sure all the dependencies exist, but it works flawlessly. I really like the ability to change the listing at each layer to your preference.

And it looks like you've got Read Mode working as well, and it works basically exactly as I'd want out of the box, which is good, because I put my plugin together in a haze and I feel like I have to relearn everything when reviewing it 😂 

Once you publish LawList to Obsidian, I'll begin sunsetting Automatic List Styles to tell people to switch to LawList.

**Comment by willem-schlieter** [comment]
@WiseGuru yeah I managed to get Read Mode done as well yesterday, so its complete now. I submitted it to Obsidian, currently waiting for the review…
